pkg/ccl/backupccl/restore_data_processor.go

```diff
@@ -276,6 +276,7 @@ func (rd *restoreDataProcessor) openSSTs(
 		readAsOfIter := storage.NewReadAsOfIterator(iter, rd.spec.RestoreTime)
 
 		cleanup := func() {
+			log.VInfof(ctx, 1, "finished with and closing %d files in span [%s-%s)", len(entry.Files), entry.Span.Key, entry.Span.EndKey)
```

Comment:
`[%s-%s)"` I nearly want to make "%+v" for a roachpb.Span do this by default as it is often much easier to read than the default string format for spans.

Reply:
Yeah, the cognitive overhead I find I encounter trying to read the clever/"concise" default printing is real (and if the key contains dashes or negative signs? game over)